rules/windows/credential_access_lsass_memdump_handle_access.toml

```
@@ -0,0 +1,118 @@
+name = "LSASS Memory Dump Handle Access"
+note = """## Triage and analysis.
+
+### Investigating
+
+Local Security Authority Server Service (LSASS) is a process in Microsoft Windows operating systems that is responsible
+for enforcing the security policy on the system. It verifies users logging on to a Windows computer or server, handles
+password changes, and creates access tokens.
+
+Adversaries may attempt to access credential material stored in the process memory of the LSASS. After a user logs on,
+the system generates and stores a variety of credential materials in LSASS process memory. This is meant to facilitate
+single sign-on (SSO) ensuring a user isn’t prompted each time resource access is requested. These credential materials
+can be harvested by an adversary using administrative user or SYSTEM privileges to conduct Lateral Movement using
+[Use Alternate Authentication Material](https://attack.mitre.org/techniques/T1550/).
+
+#### Possible investigation steps:
+
+- Validate the correct install path for the process that triggered this detection
+- Confirm that any AV or EDR solutions that trigger this detection have the correct install path
+
+### False Positive Analysis
+
+- There should be very few if any false positives for this rule. However, it may be tripped by AV or EDR solutions.
+
+### Response and Remediation
+
+- Initiate the incident response process based on the outcome of the triage
+- In case of specific credentials were compromised:
+    - Reset the password for the accounts
+
+## Config
+
+Ensure advanced audit policies for Windows are enabled, specifically
+Object Access policies [Event ID 4656](https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4656) (Handle to an Object was Requested)
+
+```
+Computer Configuration >
+Policies >
+Windows Settings >
+Security Settings >
+Advanced Audit Policies Configuration >
+System Audit Policies >
+Object Access >
+Audit File System (Success,Failure)
+Audit Handle Manipulation (Success,Failure)
+```
+
+Also, this event generates only if the object’s [SACL](https://docs.microsoft.com/en-us/windows/win32/secauthz/access-control-lists) has the required ACE to handle the use of specific access rights.
+"""
+references = [
+    "https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4656",
+    "https://twitter.com/jsecurity101/status/1227987828534956033?s=20",
+    "https://attack.mitre.org/techniques/T1003/001/",
+    "https://threathunterplaybook.com/notebooks/windows/06_credential_access/WIN-170105221010.html",
+    "http://findingbad.blogspot.com/2017/"
+]
+risk_score = 73
+rule_id = "208dbe77-01ed-4954-8d44-1e5751cb20de"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+any where event.action == "File System" and event.code == "4656" and
+
+    winlog.event_data.ObjectName: (
```

small nit for consistency
```suggestion
    winlog.event_data.ObjectName : (
```